src/test/java/com/redhat/darcy/webdriver/ChromeBrowserFactoryTest.java

```diff
@@ -0,0 +1,31 @@
+package com.redhat.darcy.webdriver;
+
+import com.redhat.darcy.webdriver.internal.CachingTargetLocator;
+import com.redhat.darcy.webdriver.internal.ForwardingTargetedWebDriver;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import static org.hamcrest.Matchers.instanceOf;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assume.assumeNotNull;
+
+public class ChromeBrowserFactoryTest {
+    @Before
+    public void checkForDriver() {
+        assumeNotNull(System.getProperty("webdriver.chrome.driver"));
+    }
+
+    @Test
+    public void shouldBeInstanceOfUntargetedChromeDriver() {
+        WebDriverBrowserFactory browserFactory = new ChromeBrowserFactory();
+
+        WebDriverBrowser browser = (WebDriverBrowser) browserFactory.newBrowser();
+        ForwardingTargetedWebDriver webDriver = (ForwardingTargetedWebDriver) browser.getWrappedDriver();
+        CachingTargetLocator targetLocator = (CachingTargetLocator) webDriver.getTargetLocator();
```

Comment:
It'd be better if you didn't have to use explicit casts here, in case the implementations ever change, but I'm not exactly sure how to deal with that right away so I think for now this is fine but something to think about.